msal-dotnet-articles/contribute/prerequisites-to-build.md

```diff
@@ -1,53 +1,48 @@
 ---
 title: Prerequisites to build MSAL.NET
+description: "Requirements to build MSAL.NET on your local computer."
 ---
 
 # Prerequisites to build MSAL.NET
 
-## Components required to build the library
-
-The following are instructions to setup Visual Studio (Mac) to build various solution files on Windows and Mac platform.
+The following are instructions to setup Visual Studio to build various MSAL.NET solution files on Windows and Mac platforms.
 
 ## Windows
 
-### Minimal Visual Studio 2019 installation
-
-* Install or update Visual Studio 2019 (any edition: Community, Pro, or Enterprise) with the following workloads:
+### Minimal Visual Studio installation
 
+* Install or update Visual Studio 2022. Any edition, such as Community, Pro, or Enterprise will work.
```

Comment:
I wasn't able to build MSAL when I only had VS 22 installed, since we still target net45. I had to install VS 19 just so that net45 SDK gets installed.